### PR TITLE
2nd Assignment by 202400730 (오경민)

### PR DESCRIPTION
**Pull request by okm0408**
# Assignment Submission

## Student Information
- **Student ID**: [Your Student ID]
- **Name**: [Your Full Name]
- **Assignment**: [Assignment Number]

## Changed Files

```
submissions
└── 202400730
    └── assignment2
        ├── problem1
        │   ├── README.md
        │   └── main.py
        ├── problem2
        │   ├── README.md
        │   └── textops
        │       ├── __init__.py
        │       ├── clean
        │       │   ├── __init__.py
        │       │   └── filters.py
        │       ├── main.py
        │       └── tokenize
        │           ├── __init__.py
        │           └── word.py
        ├── problem3
        │   ├── README.md
        │   └── main.py
        ├── problem4
        │   ├── README.md
        │   └── dsops
        │       ├── __init__.py
        │       ├── main.py
        │       ├── split
        │       │   ├── __init__.py
        │       │   └── train_test.py
        │       └── stats
        │           ├── __init__.py
        │           └── labels.py
        ├── problem5
        │   ├── README.md
        │   └── cachekit
        │       ├── __init__.py
        │       └── main.py
        └── problem6
            ├── README.md
            └── main.py
```

### Problem 1
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 2
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 3
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 4 (Optional)
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 5 (Optional)
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 6 (Optional)
- [ ] Implemented required functionality
- [ ] All tests pass locally

## Testing
- [ ] I have run `python grader.py` locally
- [ ] All implemented problems pass their tests
- [ ] I have tested edge cases mentioned in the problem statements

## Additional Notes

---
**Note**: This PR will be automatically validated for:
- Correct directory structure
- Valid Python syntax
- PR title format: `Nth Assignment by {student_id} (Full Name)`
- File organization compliance

### CODE REVIEW

File: submissions/202400730/assignment2/problem2/textops/tokenize/word.py [hunk 1]
```diff
@@ -3,12 +3,8 @@ def word_tokens(s: str) -> list[str]:
...
     Assume `s` is already normalized by clean_text.
     """
-    # TODO: 구현하세요
-    # 힌트:
-    # 1) 빈 문자열이나 공백만 있는 경우 체크: if not s or s.strip() == ""
-    # 2) 단일 공백으로 분할: s.split(" ")
-    # 3) 빈 리스트 반환 조건 잊지 말기
-    raise NotImplementedError
+    
+    return s.split(" ")
```
Comment (SH-hufs) [inline submissions/202400730/assignment2/problem2/textops/tokenize/word.py:7]: 요구사항 중에 `Split on single spaces; empty/whitespace-only -> [].`
이 부분에 대한 코드가 아직 재현이 안 되어 있습니다. 
힌트 처럼 `if not s or s.strip() ==`을 추가하고 빈 리스트를 반환해야 할 거 같아요!

Comment (okm0408) [reply submissions/202400730/assignment2/problem2/textops/tokenize/word.py:12]: 수정했습니다~ 감사합니다
